config/kcp/kustomization.yaml

```diff
@@ -1,32 +1,40 @@
 # Adds namespace to all resources.
-namespace: integration-service-system
+# namespace: integration-service-system
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: integration-service-
+# namePrefix: integration-service-
 
 # Labels to add to all resources and selectors.
 #commonLabels:
 #  someName: someValue
 
 bases:
-  - ../crd
+  # - ../crd
   - ../manager
-  - ../rbac
+  - ../kcp-rbac
 
 resources:
   - apiexport_integration.yaml
   - apiresourceschema_integration.yaml
 
 patches:
 # Needs to use default service account name right now
+# - patch: |-
```

Comment:
@dirgim @johnbieren Remove this patch because I guess it can fix the issue in this comment from @Michkov https://github.com/redhat-appstudio/infra-deployments/pull/821#pullrequestreview-1140241007, is it?

Reply:
Yes, it looks that way to me